<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">*{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0.882, y2:0.198864, stop:0 rgba(114, 211, 245, 255), stop:1 rgba(160, 240, 231, 255));
	border: none;
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">*{
	background: rgb(30, 30, 30);
	border-radius: 1px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">*{
	color: white;
	font-size: 14px;	
	font-family: Noto Sans;
}</string>
               </property>
               <property name="text">
                <string>KeyCroc - version 2.0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="maximumSize">
             <size>
              <width>85</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QPushButton" name="minimizeButton">
               <property name="maximumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">*{
	background-color: rgb(84, 255, 117);
	border-radius: 7px;
	border: none;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="maximizeButton">
               <property name="maximumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">*{
	background-color: rgb(255, 152, 55);
	border-radius: 7px;
	border: none;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="exitButton">
               <property name="maximumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">*{
	background-color: rgb(255, 55, 55);
	border-radius: 7px;
	border: none;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">*{
	background: none;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>475</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">*{
	background-color: rgb(30, 30, 30);
	border-radius: 7px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_7">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QFrame" name="frame_9">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>45</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="styleSheet">
                      <string notr="true">*{
	color: rgb(53, 133, 247);
	font-family: Noto Sans;
	font-size: 18px;
	font-weight: 300;
}</string>
                     </property>
                     <property name="text">
                      <string>Création de votre keylogger</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_10">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="spacing">
                    <number>10</number>
                   </property>
                   <property name="leftMargin">
                    <number>10</number>
                   </property>
                   <property name="topMargin">
                    <number>5</number>
                   </property>
                   <property name="rightMargin">
                    <number>10</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="intervalInput">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">*{
	background: rgb(45, 45, 45);
	color: rgb(210, 210, 210);
	padding-left: 10px;
}</string>
                     </property>
                     <property name="placeholderText">
                      <string>Interval d'envoie des logs</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="smtpInput">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">*{
	background: rgb(45, 45, 45);
	color: rgb(210, 210, 210);
	padding-left: 10px;
}</string>
                     </property>
                     <property name="placeholderText">
                      <string>Adresse du serveur SMTP</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="emailInput">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">*{
	background: rgb(45, 45, 45);
	color: rgb(210, 210, 210);
	padding-left: 10px;
}</string>
                     </property>
                     <property name="placeholderText">
                      <string>Email d'envoie des logs</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="passwordInput">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">*{
	background: rgb(45, 45, 45);
	color: rgb(210, 210, 210);
	padding-left: 10px;
}</string>
                     </property>
                     <property name="placeholderText">
                      <string>Mot de passe de l'adresse email</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="urlInput">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">*{
	background: rgb(45, 45, 45);
	color: rgb(210, 210, 210);
	padding-left: 10px;
}</string>
                     </property>
                     <property name="placeholderText">
                      <string>URL de report</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_11">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>75</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_8">
                      <item>
                       <widget class="QPushButton" name="validButton">
                        <property name="minimumSize">
                         <size>
                          <width>250</width>
                          <height>35</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>250</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">*{
	background: rgb(53, 133, 247);
	color: white;
}</string>
                        </property>
                        <property name="text">
                         <string>PushButton</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_8">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">*{
	background-color: rgb(53, 133, 247);
	border-radius: 7px;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
